packages/components/src/internal/components/ontology/OntologyLookupOptions.tsx

```diff
@@ -66,7 +69,17 @@ export class OntologyLookupOptions extends PureComponent<Props, State> {
     };
 
     onFieldChange = (evt: any): void => {
-        this.onChange(evt.target.id, evt.target.value);
+        const val = evt.target.value;
+        let changes = List<IFieldChange>([{id: evt.target.id, value: evt.target.value }]);
+
+        const valFieldIdx = this.props.domainFields.findIndex(df => df.name === val);
+        if (valFieldIdx > 0) {
+            const valFieldInsertId = createFormInputId(DOMAIN_FIELD_SHOWNININSERTVIEW, this.props.domainIndex, valFieldIdx);
+            const valFieldUpdateId = createFormInputId(DOMAIN_FIELD_SHOWNINUPDATESVIEW, this.props.domainIndex, valFieldIdx);
+            changes = changes.push({id:valFieldInsertId, value:false}, {id:valFieldUpdateId, value: false});
```

Comment:
Nice! I think this implementation works well.

Reply:
Do you think we need to handle the case for the field name that was "unset" as the ontology helper column? For example, if I pick "field A" and then change it to "field B", do I care that "field A" still has these changes? I assume no...that would get pretty tricky to keep track of.

Reply:
As you say this gets complicated very fast and I don't think trying to track it adds much value for the effort...
So my inclination is to see if we get feedback on it.

Reply:
Sounds good. Thanks.